### PR TITLE
update base64-js version. follow up #6961

### PR DESCRIPTION
**Pull request by zxcpoiu**
**Motivation**
This is originally a follow up of PR #6327 by @dewe and @jeremyong
And @mkonicek indicated this should split into a separate PR.

Now @philikon takes over #6327 as #6961

**Test Plan:**
According to #6961 and #6889
Load WebSocketExample in UIExplorer app and start websocket test server script and test sending binary data on both iOS and Android

### CODE REVIEW

**Comment by christopherdro** [comment]
@mkonicek Letting you merge this in since you guys update the internal fb npm cache manually.

**Comment by zxcpoiu** [comment]
note: this PR now depends on #6961 
and needs @philikon to review

**Comment by philikon** [comment]
> note: this PR now depends on #6961 

I don't see why that's the case.

File: package.json [hunk 1]
```diff
@@ -138,7 +138,7 @@
     "babel-register": "^6.6.0",
     "babel-types": "^6.6.4",
     "babylon": "^6.6.4",
-    "base64-js": "^0.0.8",
+    "base64-js": "^1.0.4",
```
Comment (philikon) [inline package.json]: Why not `^1.1.0` as that version seems to have a bunch of improvements over `1.0.4` (whereas `1.1.1` and `1.1.2` seem to only be about aesthetics).

**Comment by philikon** [comment]
Pro-tip: make sure you explain the motivation for your PR. You're currently not giving any, instead pointing to other PRs which also don't give motivation. If you want to make it easier for reviewers, and quicker for you, I suggest editing your PR note to include a small note about *why* you want this change. Also please don't forget a test plan.

**Comment by zxcpoiu** [comment]
@philikon 
thanks for your tips. update as you described.

**why this PR exist and described as a follow up of #6961 are because:**

1. original author @dewe add in his commit da084b79a9036fbd03c6f91a77e1ac4b65343edf in #6327 
2. later #6327 hangs for a while and everybody seems busy, so I just helped to complete the rest based on @dewe 's patch --- this PR.
3. then @philikon takes over whole PR #6327  into #6961 

**This PR should be closed if no one needed**

1. my motivation is purely hope awesome PRs won't hang. 
2. follow up author @philikon in #6961 confirmed #6961 works on bas64-js 0.0.8 ( not necessary to upgrade )

If no one thinks this is necessary, then this PR should be closed.
How do you think @dewe ?

thanks again for all the great jobs.

**Comment by dewe** [comment]
I can't recall why this was upgrade was needed. Since @philikon is handling things in the other PRs, this one should definitely be closed.

**Comment by zxcpoiu** [comment]
Thanks @dewe 
#6961 has been shipped, thank you 💯 

@mkonicek @philikon 
I think it is better to upgrade base64js to 1.x version if we using it as react native core.

do you think this is good to go?

**Comment by mkonicek** [comment]
Upgrading sounds good. I'll need to merge this manually into our internal node_modules store though.

**Comment by facebook-github-bot-8** [comment]
@cpojer would you mind taking a look at this pull request? It's been a while since the last commit was reviewed.

**Comment by javache** [comment]
@facebook-github-bot import

**Comment by facebook-github-bot-2** [comment]
Thanks for importing.

**Comment by facebook-github-bot-4** [comment]
It's been a while since the last commit was reviewed and the labels show this pull request needs review. Based on the blame information for the files in this pull request we identified @cpojer as a potential reviewer. Could you take a look please or cc someone with more context?

**Comment by mkonicek** [comment]
Took a while to get back to this one :)

@facebook-github-bot import

**Comment by facebook-github-bot-9** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3669596) to review internal test results.

**Pull request closed without merging**